packages/core/cms/faststore/content-types.json

```diff
@@ -234,6 +234,38 @@
                       }
                     }
                   }
+                },
+                "deliveryPromiseBadges": {
+                  "title": "Delivery Promise badges",
+                  "type": "object",
+                  "description": "The badges will be displayed in product cards indicating the delivery methods availability for the product.",
+                  "properties": {
+                    "enabled": {
+                      "title": "Should display Delivery Promise badges?",
+                      "type": "boolean",
+                      "default": true
+                    },
+                    "delivery": {
+                      "title": "Delivery available label",
+                      "type": "string",
+                      "default": "Available for delivery"
```

Comment:
question: I believe you already raised this question with product/design, but to confirm, it was decided to use `delivery` instead of `shipping` in this case? "Available for _delivery_"